frontend/src/components/MainContent.jsx

```diff
@@ -20,18 +28,19 @@ const MainContent = () => {
       }}
     >
       <ResponsiveNavBar />
-      <div 
+      <div
         style={{ paddingTop: '10px', overflow: 'scroll', height: '100%' }}
       >
         <Container
-          maxWidth='md'
+          maxWidth='lg'
         >
           <Routes>
             <Route path='/' element={<HomePage />} />
             <Route path='/home' element={<Navigate to={'/'} replace />} />
             <Route path='/profile' element={<ProfilePage />} />
             <Route path='/profile/:uname' element={<UserProfilePage />} />
             <Route path='/search' element={<h1>search page</h1>} />
+            <Route path='/sandwiches' element={(isAdmin || isEmployee) ? <Sandwiches /> : <NotFound />} />
```

Comment:
we should have another page for users or anons to scroll through all sandwiches in the future